modules/woocommerce-analytics/classes/wp-woocommerce-analytics-universal.php

```diff
@@ -416,4 +418,75 @@ public function get_product_categories_concatenated( $product ) {
 		return $line;
 	}
 
+	/**
+	 * Search a specific post for text content.
+	 *
+	 * Note: similar code is in a WooCommerce core PR:
+	 * https://github.com/woocommerce/woocommerce/pull/25932
+	 *
+	 * @param integer $post_id The id of the post to search.
+	 * @param string  $text    The text to search for.
+	 * @return integer 1 if post contains $text (otherwise 0).
+	 */
+	public static function post_contains_text( $post_id, $text ) {
+		global $wpdb;
+
+			// Search for the text anywhere in the post.
+		$wildcarded = "%{$text}%";
+
+		$result = $wpdb->get_var(
+			$wpdb->prepare(
+				"
+				SELECT COUNT( * ) FROM {$wpdb->prefix}posts
+				WHERE ID=%d
+				AND {$wpdb->prefix}posts.post_content LIKE %s
+				",
+				array( $post_id, $wildcarded )
+			)
+		);
+
+		return ( '0' !== $result ) ? 1 : 0;
+	}
+
+	/**
+	 * Get info about the cart & checkout pages, in particular
+	 * whether the store is using shortcodes or Gutenberg blocks.
+	 * This info is cached in a transient.
+	 *
+	 * Note: similar code is in a WooCommerce core PR:
+	 * https://github.com/woocommerce/woocommerce/pull/25932
+	 *
+	 * @return array
+	 */
+	public static function get_cart_checkout_info() {
+		$info = get_transient( 'jetpack-woocommerce-analytics-cart-checkout-info-cache' );
```

Comment:
Could you switch to using underscores instead of hyphens? It may make things easier to folks searching for all Jetpack transients with `jetpack_`

Reply:
I prefer that, makes it double-clickable for easy copy-paste too.